src/Stryker.Core/Stryker.Core/Options/Inputs/ReportFileNameInput.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using Stryker.Core.Exceptions;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public class ReportFileNameInput : Input<string>
+    {
+        protected override string Description => string.Empty;
+
+        public override string Default => null;
+
+        public string Validate()
+        {
+            if (string.IsNullOrWhiteSpace(SuppliedInput))
+            {
+                return "mutation-report";
+            }
+            if (SuppliedInput.IndexOfAny(Path.GetInvalidFileNameChars()) != -1)
```

Comment:
We should also validate that the filename does not contain an extension, since we add the extension in the reporters.

Reply:
It now throws an InvalidInputException, for any extension. This will prevent situations such as the config-filename being `mutationReport.project1`, becoming `mutationReport.project1.html`. Is this a situation we want to support?

Reply:
Good point, let's check for .json and .html for now since we know those are the extensions we are using. I am fine with InvalidInputException but I would also be fine with simply removing a .json or .html file extension during validation.

Reply:
It is updated now to match!